scylla-server-rust/src/main.rs

```diff
@@ -0,0 +1,41 @@
+use std::sync::Arc;
+
+use axum::{http::Method, routing::get, Router};
+use scylla_server_rust::{
+    controller::{
+        self, data_type_controller, driver_controller, location_controller, node_controller,
+        run_controller, system_controller,
+    },
+    prisma::PrismaClient,
+    Database,
+};
+use tower_http::cors::{Any, CorsLayer};
+
+#[tokio::main]
+async fn main() {
+    let client: Database = Arc::new(PrismaClient::_builder().build().await.unwrap());
+
+    let app = Router::new()
+        .route(
+            "/data/:dataTypeName/:runId",
+            get(controller::data_controller::get_data),
+        )
+        .route("/datatypes", get(data_type_controller::get_all_data_types))
+        .route("/drivers", get(driver_controller::get_all_drivers))
+        .route("/locations", get(location_controller::get_all_locations))
+        .route("/nodes", get(node_controller::get_all_nodes))
+        .route("/runs", get(run_controller::get_all_runs))
+        .route("/runs/:id", get(run_controller::get_run_by_id))
+        .route("/systems", get(system_controller::get_all_systems))
```

Comment:
It'd be good to separate types of routes with comments, so it's a little easier to read. If we can get this file readable, I think it's fine not to make a bunch of files for 1/2 routes each if that's not very rustlike